### ISSUE TITLE
Exception when using Chrome open file dialog

### ISSUE THREAD

**Issue by feerrenrut** [nvaccess/nvda#6139]
When using the Chrome open file dialog I encountered a non-fatal exception. I can not reproduce the issue however figure it is worth recording what we know.

This occurred while running from source after having enabled wx assertions, and fixed a known issue with `wx.CallLater` (see #6137). My branch was based off of `master` at this point:

```
* commit 28ed9628acd9c7a4272cb03e05fcbac61507bfb1
| Author: James Teh <jamie@nvaccess.org>
| Date:   Fri Jul 1 12:54:26 2016 +1000
|
|     What's New: Fix typo.
```

Focus highlight add on installed.
I was using the Google Chrome open file dialog, I was moving the review cursor and the system focus around various items. When I exited the dialog I heard the exception get raised.

I do not have steps to reproduce.

**Comment by feerrenrut** [comment]
[crashLog-nvda.log.txt](https://github.com/nvaccess/nvda/files/347109/crashLog-nvda.log.txt)

**Comment by feerrenrut** [comment]
Relevant log detail (see attachment for full log):

```
DEBUGWARNING - watchdog._watcher (04:48:10):
Trying to recover from freeze, core stack:
...  trace abridged
  File "C:\work\nvda\source\NVDAObjects\__init__.py", line 188, in findBestAPIClass
    if possibleAPIClass.kwargsFromSuper(kwargs,relation=relation):
  File "C:\work\nvda\source\NVDAObjects\UIA\__init__.py", line 392, in kwargsFromSuper
    UIAElement=UIAHandler.handler.clientObject.ElementFromHandleBuildCache(windowHandle,UIAHandler.handler.baseCacheRequest)

ERROR - eventHandler.executeEvent (04:48:10):
error executing event: mouseMove on <NVDAObjects.Dynamic_DialogIAccessibleWindowNVDAObject object at 0x0564DB70> with extra args of {'y': 834, 'x': 1406}
Traceback (most recent call last):
... trace abridged
  File "NVDAObjects\__init__.py", line 188, in findBestAPIClass
    if possibleAPIClass.kwargsFromSuper(kwargs,relation=relation):
  File "NVDAObjects\UIA\__init__.py", line 392, in kwargsFromSuper
    UIAElement=UIAHandler.handler.clientObject.ElementFromHandleBuildCache(windowHandle,UIAHandler.handler.baseCacheRequest)
COMError: (-2147024809, 'The parameter is incorrect.', (None, None, None, 0, None))
```

**Comment by ehollig** [comment]
@feerrenrut is this still applicable after the upgrade to WX?

**Comment by Adriani90** [comment]
@feerrenrut, @jcsteh is this still an issue with WX Python 4.0.4? Or can it be closed? Thanks.

**Comment by Adriani90** [comment]
@feerrenrut this might be fixed in WX python 4.0.4. Are you still encountering this issue?
cc: @josephsl

**Comment by feerrenrut** [comment]
I can't reproduce this, I'm going to close the issue.
